LICENSE-3rdparty.csv

```diff
@@ -29,6 +29,7 @@ require,url-parse,MIT,Copyright 2015 Unshift.io Arnout Kazemier the Contributors
 require,whatwg-fetch,MIT,Copyright 2014-2016 GitHub Inc.
 dev,@babel/core,MIT,Copyright 2014-present Sebastian McKenzie and other contributors
 dev,@babel/preset-env,MIT,Copyright 2014-present Sebastian McKenzie and other contributors
+dev,@elastic/elasticsearch,Apache license 2.0,Copyright 2013 Elasticsearch B.V.
```

Comment:
I'm not 100% sure it matters, but just to be safe we usually write this license as `Apache-2.0`.